package.json

```diff
@@ -56,7 +56,6 @@
     "modulePathIgnorePatterns": [
       "/^((?!src).)/"
     ],
-    "testResultsProcessor": "<rootDir>/node_modules/ts-jest/coverageprocessor.js",
```

Comment:
this is making me not able to run `jest` is it necessary to keep?

Reply:
nope